### PR TITLE
dlb:Fix CI errors 

### PR DESCRIPTION
**Pull request by hj-johannes-lee**
dlb:Fix CI errors

Fix linter errors by gofmt -s
Fix building image errors by putting image name in ci.yaml file

Signed-off-by: Hyeongju Johannes Lee <hyeongju.lee@intel.com>

### CODE REVIEW

**Review by mythi — COMMENTED** [review]
LGTM

**Comment by bart0sh** [comment]
@hj-johannes-lee Please change name and description of this PR (and commit) to reflect the reality. Remember, it's going to be merged into the main PR branch. We don't want 2 commits with the same name there.

**Review by bart0sh — CHANGES_REQUESTED** [review]
^^^^

**Comment by mythi** [comment]
> We don't want 2 commits with the same name there.

@bart0sh you don't plan to squash this into the original commit (in #4)?

**Comment by hj-johannes-lee** [comment]
Yeah, I was thinking that you would just squash this into the original,, so I just copied.! If not, tell me!

**Comment by bart0sh** [comment]
I don't want to squash anything that looks logical. But even if I want to do that I'd anyway prefer commits named and described adequately. For example, this PR (and commit) is called "dlb: Add new device plugin", but doesn't add any new device plugin.

**Comment by mythi** [comment]
That I agree with. f09a72f0ee7bac81c5b1dc863ff02a2af2232ac0 commit message should be rewritten

**Comment by hj-johannes-lee** [comment]
I see.! Let me rewrite the message.1

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -85,7 +86,7 @@ jobs:
           - sgx-aesmd-demo
           - ubuntu-demo-openvino
           - dlb-dpdk-demo
-          - dlb-libdlb-demo
+          - dlb-libdlb-demo
```
Comment (bart0sh) [inline .github/workflows/ci.yaml:89]: Please, remove trailing spaces.

**Review by bart0sh — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (hj-johannes-lee) [reply .github/workflows/ci.yaml]: I immediately noticed after pushing it. Idk.. I think my visual code in linux laptop sometimes malfunctions..
I saw +2/-1 but the git log showed only one line I added for intel-dlb-plugin..! And,, after I pushed it suddenly showed that.. I don't understand.. but I changed (hoping that you would not see before I push it.. but you already saw it.!)

**Review by bart0sh — APPROVED** [review]
/lgtm

**Pull request merged**